modules/rosa-required-aws-service-quotas.adoc

```diff
@@ -6,7 +6,11 @@
 [id="rosa-required-aws-service-quotas_{context}"]
 = Required AWS service quotas
 
-The table below describes the AWS service quotas and levels required to create and run an {product-title} cluster.
+The table below describes the AWS service quotas and levels required to create and run one {product-title} cluster. While most default values are suitable for most workloads, you are likely to need to request additional quota for the following cases:
+
+* ROSA clusters require at least 100 vCPUs, but the default maximum value for vCPUs assigned to Running On-Demand Standard Amazon EC2 instances is `5`. Therefore if you have not created a ROSA cluster using the same AWS account previously, you must request additional EC2 quota for `Running On-Demand Standard (A, C, D, H, I, M, R, T, Z) instances`.
```

Comment:
It's unclear to me why the last part here is in backticks. Is this some literal value somewhere (CLI/UI)? I don't see any references of anything like that anywhere.

Reply:
@bergerhoffer It's a label / field and a specific quota type in the AWS Console, which I think is best kept as a literal value.